library/src/test/java/com/cottacush/android/currencyedittext/CurrencyRounderTest.kt

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright (c) 2019 Cotta & Cush Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cottacush.android.currencyedittext
+
+import org.junit.Assert
+import org.junit.Test
+
+class CurrencyRounderTest {
```

Comment:
This test class seems too long. Can you check if you can reduce it this [parameterized-tests](https://github.com/junit-team/junit4/wiki/parameterized-tests)?

Reply:
checking this out.